operations/rates.md

```diff
@@ -366,3 +485,19 @@ Note that prices are defined daily, so when the server receives the UTC interval
 ```javascript
 {}
 ```
+
+#### Add Rate type
```

Comment:
I notice that `Add Rate type` differs from `Rate type` in that the former does not include `AvailabilityBlock`. Should we say in the description for `Add rates` that you cannot add availability blocks with this operation, or is that not necessary?

Reply:
I think we can mention it since it can prevent confusion.